Refresh.GameServer/Endpoints/Game/Levels/LevelEndpoints.cs

```diff
@@ -29,7 +29,7 @@ public class LevelEndpoints : EndpointGroup
         (int skip, int count) = context.GetPageData();
 
         DatabaseList<GameLevel>? levels = categoryService.Categories
-            .FirstOrDefault(c => c.GameRoute.StartsWith(route))?
+            .FirstOrDefault(c => c.GameRoutes.Any(x => x.StartsWith(route)))?
```

Comment:
We usually use the first letter of the name instead of `x`, so `r` for `route` in this case.